app/views/assessment360/all_students_all_reviews.html.erb

```diff
@@ -1,9 +1,37 @@
+<script>
```

Comment:
Add a comment saying that the code below provides blanks for typing in search criteria.